### PR TITLE
openjpeg, bump version

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: media-libs/openjpeg/openjpeg-2.2.0.recipe [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+	openjpeg${secondaryArchSuffix}_devel = $portVersion
+	devel:libopenjp2$secondaryArchSuffix = 7.0.2 compat >= 7
+	"
+REQUIRES_devel="
+	openjpeg$secondaryArchSuffix == $portVersion
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libpng$secondaryArchSuffix
```
Comment (korli) [inline media-libs/openjpeg/openjpeg-2.2.0.recipe]: Should be devel:libpng16

Comment (Begasus) [inline media-libs/openjpeg/openjpeg-2.2.0.recipe]: IIRC it wouldn't be needed for the devel package? I can change it, no prob (but I think I just left it as it was with the previous recipe) ...

Comment (korli) [inline media-libs/openjpeg/openjpeg-2.2.0.recipe]: Yes, it's a leftover to fix. devel:libpng will be removed eventually.

Comment (Begasus) [inline media-libs/openjpeg/openjpeg-2.2.0.recipe]: Ok, will check the build first with devel:libpng16 :)

**Comment by korli** [comment]
I would leave openjpeg-2.1.2.recipe as is when it still works for x86_gcc2 though.

**Comment by Begasus** [comment]
added v2.1.2 back into the PR

**Pull request merged**